### PR TITLE
Added phpunit as a dependency to run tests independently

### PR DESCRIPTION
**Pull request by francescomalatesta**
As the title says :)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,5 +23,8 @@
         "psr-4": {
             "Ae\\FeatureBundle\\": ""
         }
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^5.1"
```
Comment (EmanueleMinotto) [inline composer.json]: This bundle is compatible with PHP 5.4, so to prevent a dependency hell please add PHPUnit `^4.8` (afaik still maintained)

Comment (francescomalatesta) [inline composer.json]: :+1:

File: composer.json [hunk 2]
```diff
@@ -23,5 +23,8 @@
         "psr-4": {
             "Ae\\FeatureBundle\\": ""
         }
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^4.8"
```
Comment (EmanueleMinotto) [inline composer.json]: `"phpunit/phpunit": "^4.8|^5.1"` (always for the hell dependency)

Comment (francescomalatesta) [inline composer.json]: :+1:  :+1:

**Pull request merged**